src/ui/DynamicFacet/DynamicFacetBreadcrumbs.ts

```diff
@@ -14,7 +14,7 @@ export class DynamicFacetBreadcrumbs {
   }
 
   private create() {
-    this.element = $$('ul', { className: 'coveo-dynamic-facet-breadcrumb coveo-breadcrumb-item' }).el;
+    this.element = $$('div', { className: 'coveo-dynamic-facet-breadcrumb coveo-breadcrumb-item' }).el;
```

Comment:
Similar to JF's comment, wondering if this change could break certain styles, hmm